core/templates/domain/utilities/url-interpolation.service.ts

```diff
@@ -132,10 +132,10 @@ export class UrlInterpolationService {
       return null;
     }
 
-    // Valid pattern: <alphanum>
+    // Valid pattern: <alphanum>.
```

Comment:
Ditto, I don't think we need `.` here as well.

Reply:
Done!